content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/README.md

```diff
@@ -63,6 +63,28 @@ data-cmp-hook-carousel="next"
 data-cmp-hook-carousel="indicator"
 ```
 
+### Enabling Carousel Editing Functionality
+The following properties and child nodes are required to enable full editing functionality for the Carousel:
+
+1. `./cq:isContainer` - set to `true`, marks the Carousel as a container component
+2. `./cq:editConfig` - `afterchilddelete`, `afterchildinsert` and `afterchildmove` listeners should be provided via
+the edit configuration of the proxy. `_cq_editConfig.xml` contains the recommended actions and can be copied to the proxy component.
+
+In addition, a message request handler should be registered in the site Client Library:
```

Comment:
I would rearrange a bit this part as it sounds now that it's mandatory to provide a message handler (although people could use the builtin one). I would start with something like:

> The Carousel site Client Library should be added to the page. It provides a handler for message requests between the editor and the Carousel. In case the builtin Client Library is not added, a message request handler should be registered ...